package.json

```diff
@@ -9,7 +9,7 @@
     "test": "npm run lint",
     "prepush": "npm test",
     "postinstall": "install-app-deps",
-    "pack": "npm run build && build --dir && babel --no-comments --compact --minified --out-file app/dist/bundle.js app/dist/bundle.js",
+    "pack": "npm run build && build --dir && BABEL_ENV=production babel --out-file app/dist/bundle.js --no-comments --minified app/dist/bundle.js",
```

Comment:
@rauchg just tested this on Windows and i am getting : 
`'BABEL_ENV' is not recognized as an internal or external command, operable program or batch file.`

I added cross-env and i was able to build it like that. Do you think there is a better option for this ?